file/pkg/service/service.go

```diff
@@ -80,3 +82,55 @@ func (f *fileService) ListFileShare(ctx context.Context, in *pb.ListFileShareReq
 	log.Debugf("Listed File Share successfully, #num=%d, fileshares: %+v\n", len(fileshares), fileshares)
 	return nil
 }
+
+func (f *fileService) GetFileShare(ctx context.Context, in *pb.GetFileShareRequest, out *pb.GetFileShareResponse) error {
+	log.Info("Received GetFileShare request.")
+
+	fs, err := db.DbAdapter.GetFileShare(ctx, in.Id)
+	if err != nil {
+		log.Errorf("failed to get fileshare: %v\n", err)
+		return err
+	}
+	var tags []*pb.Tag
+	for _, tag := range fs.Tags {
+		tags = append(tags, &pb.Tag{
+			Key:   tag.Key,
+			Value: tag.Value,
+		})
+	}
+	out.Fileshare = &pb.FileShare{
+		Id:                 fs.Id.Hex(),
+		CreatedAt:          fs.CreatedAt,
+		UpdatedAt:          fs.UpdatedAt,
+		Name:               fs.Name,
+		Description:        fs.Description,
+		TenantId:           fs.TenantId,
+		UserId:             fs.UserId,
+		BackendId:          fs.BackendId,
+		Backend:            fs.Backend,
+		Size:               fs.Size / utils.GB_FACTOR,
+		Type:               fs.Type,
+		Status:             fs.Status,
+		Region:             fs.Region,
+		AvailabilityZone:   fs.AvailabilityZone,
+		Tags:               tags,
+		Protocols:          fs.Protocols,
+		SnapshotId:         fs.SnapshotId,
+		Encrypted:          fs.Encrypted,
+		EncryptionSettings: fs.EncryptionSettings,
+		Metadata:           fs.Metadata,
+	}
+	log.Info("Get file share successfully.")
+	return nil
+}
+
+func (f *fileService) DeleteFileShare(ctx  context.Context, in *pb.DeleteFileShareRequest, out *pb.DeleteFileShareResponse) error {
+	log.Info("Received DeleteFileShare request.")
+	err := db.DbAdapter.DeleteFileShare(ctx, in.Id)
```

Comment:
Seems only delete the recorde from database, not delete it from the cloud backend, need to take care of it.

Reply:
@sfzeng According to current plan. Delete API should delete the FS object from multi-cloud FS db. 
The FS created can be in usage through different mount targets. So deleting FS at cloud backend is not a good idea before make these checks like the usage of FS.

Reply:
ok, but if we delete it without considering the cloud backend, then the user may cannot check and manage it anymore. Maybe we can open an issue to trace it.

Reply:
I created the issue: https://github.com/sodafoundation/multi-cloud/issues/1022

Reply:
Agree. When we add more operations like Mounting and mount target, then we can delete from backednd too. Thanks @sfzeng for opening issue